layouts/partials/schedule-consultation.html

```diff
@@ -0,0 +1,10 @@
+<section class="py-9">
```

Comment:
This was moved from case-studies/list.html to a partial file. No other changes were made.